app/src/main/java/com/gathering/android/event/eventdetail/EventDetailComposable.kt

```diff
@@ -0,0 +1,214 @@
+            .fillMaxSize()
+    ) {
+        EventImage(photoUrl)
+        EventInfo(
+            eventName = eventName,
+            eventHostEmail = eventHostEmail,
+            description = description,
+            address = address,
+            date = date, time = time,
+        )
+        AttendeeListButton(onAttachListClicked)
+        AcceptTypeAttendee(onYesButtonClick, acceptType, onNoButtonClick, onMaybeButtonClick)
+    }
+}
+
+@Composable
+fun EventImage(imageUrl: String) {
+    Card {
+        val painter = if (imageUrl.isEmpty()) {
+            painterResource(id = R.drawable.ic_launcher_foreground)
+        } else {
+            rememberAsyncImagePainter(model = imageUrl)
+        }
+        Image(
+            painter = painter,
+            contentScale = ContentScale.Crop,
+            contentDescription = "EventPhoto",
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(200.dp)
+        )
+    }
+}
+
+@Composable
+private fun EventInfo(
+    eventName: String,
+    eventHostEmail: String,
+    description: String,
+    address: String,
+    date: String,
+    time: String
+) {
+    EventTitle(eventName)
+    TextInfo(eventHostEmail)
+    TextInfo(description)
+    TextInfo(address)
+    TextInfo(date)
+    TextInfo(time)
+}
+
+@Composable
+fun EventTitle(eventName: String) {
+    Text(
+        fontSize = 20.sp,
+        fontWeight = FontWeight.Bold,
+        text = eventName,
+        modifier = Modifier.padding(start = 10.dp, 40.dp)
+    )
+}
+
+@Composable
+fun TextInfo(text: String) {
+    Text(
+        fontSize = 16.sp,
+        text = text,
+        modifier = Modifier.padding(10.dp)
+    )
+}
+
+@Composable
+fun AttendeeButton(
+    text: String,
+    state: Boolean,
+    onClick: () -> Unit,
+) {
+    val backgroundColor = if (state) Color.Gray else Color.DarkGray
+    Button(
+        shape = RoundedCornerShape(0.dp),
+        onClick = { onClick() },
+        modifier = Modifier
+            .height(60.dp)
+            .width(120.dp)
+            .padding(5.dp)
+            .background(backgroundColor),
+        colors = ButtonDefaults.buttonColors(containerColor = backgroundColor),
+        contentPadding = PaddingValues(8.dp),
+        content = {
+            Text(
+                text = text,
+                modifier = Modifier.padding(4.dp),
+            )
+        },
+    )
+}
+
+
+@Composable
+fun AttendeeListButton(onClick: () -> Unit) {
+    Button(
+        colors = ButtonDefaults.buttonColors(
+            containerColor = Color.Transparent,
+            contentColor = Color.DarkGray
+        ),
+        onClick = onClick,
+        content = {
+            Text(
+                fontWeight = FontWeight.Bold,
+                fontSize = 20.sp,
+                text = "Click to see attendee",
+            )
+        }
+    )
+}
+
+@Composable
+private fun AcceptTypeAttendee(
+    onYesButtonClick: () -> Unit,
+    acceptType: AcceptType,
```

Comment:
Please reorder the `acceptType` and `onYesButton`, put all higher order functions near each other.